scripts/latest-dataset-to-csv/index.js

```diff
@@ -3,8 +3,9 @@ const moment = require('moment');
 const createCsvWriter = require('csv-writer').createObjectCsvWriter;
 
 const csvWriter = createCsvWriter({
+  alwaysQuote: true,
```

Comment:
Necessary to work around a bug in csv-writer where it isn't auto-quoting files that contain CR but not LF

This is fixed in csv-writer - see https://github.com/ryu1kn/csv-writer/pull/57

So we could pick up a release of csv-writer that has the fix, and then remove this line